src/senselab/audio/tasks/preprocessing/preprocessing.py

```diff
@@ -103,37 +103,105 @@ def select_channel_from_audios(audios: List[Audio], channel_index: int) -> List[
     return mono_channel_audios
 
 
-def chunk_audios(data: List[Tuple[Audio, Tuple[float, float]]]) -> List[Audio]:
```

Comment:
@ibevers why did you replaced `chunk_audios` with `extract_segments`? they don't do they same thing. `extract_segments` may use `chunk_audios` under the hood.

Reply:
With chunk audios, I don't immediately know whether two extracted chunks are from the same audio file, whereas I do with `extract_segments`. I updated `extract_segments` to use `chunk_audios`.